build.gradle

```diff
@@ -227,6 +234,28 @@ subprojects {
                 compileClasspath += generated.output
                 runtimeClasspath += generated.output
             }
+            scala {
+                compileClasspath += generated.output
+                runtimeClasspath += generated.output
+            }
+            scoverage {
+                compileClasspath += generated.output
+                runtimeClasspath += generated.output
+            }
+            testScoverage {
```

Comment:
if this is even needed shouldn't it be under test?

Reply:
I swear I tried that and things broke, but I tried it again just now, and everything seems to be alright. So yes.

I guess putting it here caused the test source set to pull it in, but it's only really there.